Gemfile

```diff
@@ -1,11 +1,11 @@
 source 'https://rubygems.org'
 
-gem 'rake'
 gem 'cocoapods', '~> 1.10'
-gem 'xcpretty-travis-formatter'
 gem 'dotenv'
-gem 'xcode-install'
 gem 'fastlane', '~> 2'
+gem 'rake'
```

Comment:
Feels weird not to have version constraints on any of these. While we're reordering, WDYT about adding them?

Reply:
Good point, will add that

Reply:
Fixed in 91ccf8a